### PR TITLE
Add observability settings

### PR DESCRIPTION
**Pull request by nrfox**
Part of https://github.com/kiali/kiali/pull/4532

### CODE REVIEW

File: roles/default/kiali-deploy/defaults/main.yml [hunk 1]
```diff
@@ -311,8 +311,13 @@ kiali_defaults:
...
     gzip_enabled: true
-    metrics_enabled: true
-    metrics_port: 9090
+    observability:
+      metrics:
+        enabled: true
+        port: 9090
+      tracing:
+        collector_url: http://jaeger-collector.istio-system:14268/api/traces
+        enabled: false
```
Comment (pbajjuri20) [inline roles/default/kiali-deploy/defaults/main.yml:320]: Is enabled false (or) true?

**Review by jmazzitelli — DISMISSED** [review]
lgtm

**Comment by pbajjuri20** [comment]
tested this PR, It looks good to me.

**Review by pbajjuri20 — DISMISSED** [review]

**Comment by nrfox** [comment]
Rebased with master

**Review by jmazzitelli — APPROVED** [review]

**Pull request merged**